### PR TITLE
Fix annoying print_brute messages in ssh_login

### PR DESCRIPTION
**Pull request by wvu**
Testing `ssh_login`:

- [x] Test the module
- [x] See conservative printing (no "starting bruteforce" message)
- [x] Test the module with `VERBOSE` enabled
- [x] See the expected verbose messages

Testing `AuthBrute`:

- [x] Test `ssh_login` using the existing `print_brute` messages
- [x] See that they're rebuilt to the "legacy" (superior) format
- [x] Test with a message matching the newly inverted regex in `build_brute_message`
- [x] See the message isn't doubled

### CODE REVIEW

File: modules/auxiliary/scanner/ssh/ssh_login.rb [hunk 1]
```diff
@@ -17,19 +17,20 @@ class MetasploitModule < Msf::Auxiliary
...
       },
-      'Author'      => ['todb'],
+      'Author'         => ['todb'],
       'References'     =>
         [
           [ 'CVE', '1999-0502'] # Weak password
         ],
-      'License'     => MSF_LICENSE
+      'License'        => MSF_LICENSE,
+      'DefaultOptions' => {'VERBOSE' => false} # Disable annoying connect errors
```
Comment (timwr) [inline modules/auxiliary/scanner/ssh/ssh_login.rb]: isn't VERBOSE false by default anyway?

Comment (wvu-r7) [inline modules/auxiliary/scanner/ssh/ssh_login.rb]: It is in the mixin, which means you get ALL the messages. That's too much here.

Comment (timwr) [inline modules/auxiliary/scanner/ssh/ssh_login.rb]: Cool thanks

Comment (wvu-r7) [inline modules/auxiliary/scanner/ssh/ssh_login.rb]: Oh, sorry, I meant it's `true` by default because of the `AuthBrute` mixin.

**Comment by busterb** [comment]
Landable? Looks to be.

**Comment by wvu-r7** [comment]
This is ready to test and land. Thanks.

**Comment by busterb** [comment]
## Release Notes

This fixes the output of brute force modules to include proper host details in verbose logging messages.

**Comment by wvu-r7** [comment]
Thank you!

**Pull request merged**